Drupal 7 port, for great justice!

adam-vessey opened the pull request.

adam-vessey committed:
Drop in Drupal 7.28's autocomplete.

Want to be able to track the changes made to it.

adam-vessey committed:
A fair ways in?

Seems to be some differences in the way D7 caches forms, though...

adam-vessey committed:
Stop trying to be quite so clever...

adam-vessey committed:
Closer...

adam-vessey committed:
Apparently working.

adam-vessey committed:
Coding standards.

adam-vessey committed:
Get alt results working.

adam-vessey committed:
Make the hidden field work.

adam-vessey committed:
Add in CSS which was missed.

adam-vessey committed:
Merge remote-tracking branch 'base/master' into 7.x

adam-vessey committed:
Improving documentation, fixing a slight error with the "fq" spec.

adam-vessey committed:
Add in CI stuffs.

adam-vessey committed:
Fix module name.

adam-vessey committed:
Add missing dependency.

adam-vessey committed:
Fix Travis? Also, remove unused JS.

willtp87 commented on README.md:
```diff
@@ -0,0 +1,56 @@
+# Islandora Authority Module [![Build Status](https://travis-ci.org/discoverygarden/islandora_authority.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_authority)
```
That last link might need to go to discoverygarden.

willtp87 commented on build.xml:
```diff
@@ -0,0 +1,105 @@
+
+  <target name="test">
+    <exec executable="drush">
+      <arg line="test-run --uri=http://localhost --xml=${basedir}/build/test 'Islandora Manuscript'" />
```
Islandora Manuscript?

adam-vessey replied:
Guess from where this was copied. :P

willtp87 commented on .travis.yml:
```diff
@@ -0,0 +1,31 @@
+language: php
+php:
+  - 5.3.3
+  - 5.4
```
Could we add 5.5?

willtp87 commented on .travis.yml:
```diff
@@ -0,0 +1,31 @@
+  - $ISLANDORA_DIR/tests/scripts/line_endings.sh sites/all/modules/islandora_authority
+  - drush coder-review --reviews=production,security,style,i18n,potx,sniffer islandora_authority
+  - phpcpd --names *.module,*.inc,*.test sites/all/modules/islandora_authority
+#  - drush test-run --user=1 --uri=http://localhost:8081 "Islandora Manuscript"
```
Copy pasta.

willtp87 commented on js/autocomplete.js:
```diff
@@ -128,360 +65,209 @@ Drupal.settings.islandora_authority.jsAC.prototype.select = function (node) {
+Drupal.islandora_authority_jsAC.prototype.selectLeft = function () {
+  if (this.sub_selected) {
+    this.sub_unhighlight(this.sub_selected);
+    //this.highlight(this.selected);
```
Commented out code?

adam-vessey committed:
Get the process functions out of the .module.

...  Seems weird that elements cannot specify files to load in order to break
"#process" functions and the like into separate files...  Anyway.

adam-vessey committed:
Squash some commented out code.

adam-vessey committed:
s/Manuscript/Authority/g, and throw PHP 5.5 into the matrix.

adam-vessey committed:
Reliably load include, 'cause Drupal is silly.
